packages/keystone/preprocessors/schemaDoc.js

```diff
@@ -60,9 +61,11 @@ const { GQL_CUSTOM_SCHEMA_TYPE } = require('../schema')
  * @return {string}
  * TODO: localize error message
```

Comment:
It's strange that `preprocessors/schemaDoc` doesn't have code to translate errors, but `schema.graphql` and `schema.ts` already has errors translated

Reply:
I've add this feature. But I'm not sure that it's ok for docs. At the moment if you just change the translation of the error you need to regenerate schema.graphql. And really at the moment we don't have all translations keys for our errors ... Thats sad, but it's not a part of this PR.